src/lib/zod/index.ts

```diff
@@ -151,12 +151,25 @@ export const createUsersBulkSchemaInput = z
   })
   .strict();
 
+export const createOrgUsersBulkSchemaInput = z
```

Comment:
let's avoid adding things to this file unless it's really clear they're reusable